qt/aqt/webview.py

```diff
@@ -96,14 +97,74 @@ def _onCmd(self, str):
         return self._onBridgeCmd(str)
 
 
+# Add-ons
+##########################################################################
+
+
+@dataclasses.dataclass
+class WebContent:
+    """Stores all dynamically modified content that a particular web view will be
+    populated with.
+
+    Attributes:
+        body {str} -- HTML body
+        head {str} -- HTML head
+        css {List[str]} -- List of media server subpaths, each pointing to a CSS file
+        js {List[str]} -- List of media server subpaths, each pointing to a JS file
+
+    Important Notes:
+        - When modifying the attributes specified above, please make sure your changes
+        only perform the minimum requried edits to make your add-on work. You should
+        avoid overwriting or interfering with existing data as much as possible,
+        instead opting to append your own changes, e.g.:
+        
+            def on_webview_will_set_content(web_content: WebContent, context):
+                web_content.body += "<my_html>"
+                web_content.head += "<my_head>"
+                web_content.css.append("my_addon.css")
+                web_content.js.append("my_addon.js")
```

Comment:
I think having a bare filename here (and in the new hook's docs) might be a bit confusing - perhaps the css/js examples are best omitted from this example, or contain the full code below as well?

Reply:
Good point! I think removing the css/js examples is the best approach here, since they're somewhat redundant with the examples below anyway.

Reply:
ok, I've updated the examples both here and in the new hook's docs. Please let me know if there's anything further you would like me to tweak!